snippets/coreos_cloudconfig.erb

```diff
@@ -4,14 +4,44 @@ kind: snippet
 name: coreos_cloudconfig
 %>
       coreos:
-        etcd:
+        etcd2:
 <% if @host.params['etcd_discovery_url'] -%>
           discovery: <%= @host.params['etcd_discovery_url'] %>
 <% end -%>
-          addr: <%= @host.ip %>:4001
-          peer-addr: <%= @host.ip %>:7001
+          advertise-client-urls: http://<%= @host.ip %>:2379,http://<%= @host.ip %>:4001
+          initial-advertise-peer-urls: http://<%= @host.ip %>:2380,http://<%= @host.ip %>:7001
+          listen-client-urls: http://0.0.0.0:2379,http://0.0.0.0:4001
+          listen-peer-urls: http://0.0.0.0:2380,http://0.0.0.0:7001
         units:
-          - name: etcd.service
+<% subnet = @host.subnet -%>
```

Comment:
Of course :) We just wanted to push these changes in one PR for simplicity but you are right :+1: